docker-compose/IS-with-Analytics/README.md

```diff
@@ -14,9 +14,14 @@ Runs a pre configured Identity Server container and Identity Server Analytics co
 
      * [Identity Server Dockerfile](../../dockerfiles/is/README.md)
      * [Identity Server Analytics Dockerfile](../../dockerfiles/is-analytics/README.md)
-     > In the `docker-compose.yml`, remove the `dockerhub.wso2.com/` prefix from the `image` name
+     > In order to run the Docker Compose configuration, you will need an active Subscription from WSO2 since the 
```

Comment:
@ThilinaManamgoda shall we make the word 'Subscription' start with simple 's' in order to be more consistent?